### DISCUSSION TITLE
Unable to Forward Cookie to Fetch API Request

### DISCUSSION THREAD

**Discussion by zidinberu** [vercel/next.js#59260] [Help]
### Summary

I have Nextjs 14 App router app. Client components are forwarding the cookie properly during fetch API requests. However, server components are NOT. The cookie is created by my Java backend API with samesite LAX, httpOnly, secure properties. NextJS runs on www.domain.com and the API runs on api.domain.com. Please help!!! Here's what I tried:

import { cookies } from "next/headers";

const getData = async (url: string): Promise<Data> => {
     const response = await fetch(url, {
      method: "POST",
      headers: { Cookie: cookies().toString() },
    });
const SummaryCard = async () => {
  const { a, b } = await getData(
    `${process.env.NEXT_PUBLIC_API_BASE_URL}/api/accounts`
  );
  return(<div></div>)
}

**Comment by Austin1serb** [comment]
I am having the same problem:
"use server"
import { cookies } from "next/headers";

const page = async () => {

  const response = await fetch('http://localhost:8000/api/user/check-login', {
    headers: {
      Cookie: cookies().toString()
    },
  });

  const data = await response.json()



  return (
    <div>
      {data.isLoggedIn.toString()}


    </div>
  )
}

export default page
